### PR TITLE
pkg/contact: replaces kad and kadclient

### PR DESCRIPTION
**Pull request by jenlij**
What: creates pkg/contact

Why: Need a package to replace kademlia service and kademlia client

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: pkg/communication/dialer.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package kademliaclient
+package communication
```
Comment (egonelbre) [inline pkg/communication/dialer.go]: Note, for the actual PR, separate the deletion from the changes and renames. It's easier to approve deletion.

Comment (jenlij) [reply pkg/communication/dialer.go]: Ok sounds good

Comment (egonelbre) [inline pkg/communication/dialer.go]: Also, there probably is a better name than `communication` for this package.

**Comment by egonelbre** [comment]
Didn't bootstrap had other features besides bootstrapping the network?

**Pull request closed without merging**